modules/core/04-channel/types/events.go

```diff
@@ -8,11 +8,19 @@ import (
 
 // IBC channel events
 const (
-	AttributeKeyConnectionID       = "connection_id"
-	AttributeKeyPortID             = "port_id"
-	AttributeKeyChannelID          = "channel_id"
-	AttributeKeyChannelState       = "channel_state"
-	AttributeVersion               = "version"
```

Comment:
I renamed this one to stay consistent with the format of starting with `AttributeKey`.

Reply:
was this added in channel upgradability? Can't remember :sweat_smile: If not, wouldn't this be API breaking and as such not backported to 8.1?

Reply:
It's already used in `emitChannelOpenIntEvent` and `emitChannelOpenTryEvent`, so yes, it would be API breaking. That's I added a note to the migration docs.